python/cli/prelude_cli/views/partner.py

```diff
@@ -128,3 +128,13 @@ def observed_detected(controller, test_id, hours):
     with Spinner(description='Getting observed / detected stats'):
         data = controller.observed_detected(test_id=test_id, hours=hours)
     print_json(data=data)
+
+@partner.command('advisories')
+@click.argument('partner', type=click.Choice([Control.CROWDSTRIKE.name], case_sensitive=False))
+@click.pass_obj
+@handle_api_error
+def partner_advisories(controller, partner):
+    """ Get advisories provided by partner """
+    with Spinner(description='Getting partner advisories'):
+        data = controller.list_advisories(partner=Control[partner])
+    print_json(data=data)
```

Comment:
do we still want to review with FE what fields we should keep before shipping this or just plan to adjust later?

Reply:
since shipping seems imminent, i'm wondering if we should prune now, down just to the ID and then add things back in once FE (and honestly probably Pete / UX) know what they want from it

even if we leave the API out of the CLI / SDK, it's still _technically_ backwards incompatible to change the API later since its a public endpoint

Reply:
I like that idea- prune down to the bare minimum and add back as needed. Not a breaking change!

Reply:
https://github.com/preludeorg/service/pull/2095